src/Components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,27 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClose } from '@fortawesome/free-solid-svg-icons';
+import './Dropdown.scss';
+
+interface DropdownConfig {
```

Comment:
```suggestion
interface IDropdownProps {
```
Just could be little bit more clear

Reply:
Done.